### ISSUE TITLE
NUmeric values in Album Artists

### ISSUE THREAD

**Issue by henkiewie** [LMS-Community/slimserver#763]
Hi,
It seems like there are no numerical values in Album Artists. (screenshot 1)
The numbers still appear in All Artists and New Music but not the Album Artist that should be there (10cc) (screenshot 2)
I do get the artist by doing a search  (screenshot 3)
![Screenshot 2022-02-14 at 08 00 04](https://user-images.githubusercontent.com/74558874/153815269-42b722ad-0709-4f7c-9888-ab2bdc3037f6.png)
![Screenshot 2022-02-14 at 08 01 09](https://user-images.githubusercontent.com/74558874/153815373-35814823-bbcc-4c16-b0b3-b0aeb1f21864.png)
![Screenshot 2022-02-14 at 08 08 58](https://user-images.githubusercontent.com/74558874/153816341-b6df65e6-d081-47b8-ae96-17ef4aeae21a.png)
I have tried the stable build the Squeezelite-x version and the docker dev build.
Version: 8.3.0 - 1644768372 @ Sun Feb 13 17:26:41 CET 2022
Hostname: 1929aa05f888
Server IP Address: 172.17.0.14
Operating system: Debian (Docker) - EN - utf8
Platform Architecture: armv7l-linux
Perl Version: 5.32.1 - arm-linux-gnueabihf-thread-multi-64int
Audio::Scan: 1.02
IO::Socket::SSL: 2.069
Database Version: DBD::SQLite 1.58 (sqlite 3.22.0)
I tried different behavior settings they all give the same result. Changing theme to default does not help.
I tried the forum a couple of weeks ago, but no answers. until now.

**Comment by michaelherger** [comment]
I have "22-Pistepirkko" etc. in my collection. Do you see the same behaviour in Squeezeplay or the Default web skin?

**Comment by henkiewie** [comment]
Hi Michael,
Thank you for the quick reply.
Yes the behavior is the same in every client i tried. I don't know when it changed because i have been using different versions.
It just caught my eye about a month ago.
My ID3 tags are pretty clean I think.
Doing a clean scan gives me the same results.
regards,
Henk
![Screenshot 2022-02-14 at 10 09 48](https://user-images.githubusercontent.com/74558874/153834217-a9b2ecea-4a46-43ed-b218-2b61e6ac8f45.png)

**Comment by mherger** [comment]
Oh, it’s missing from the Album Artist view only… I read too quickly. I’ll double check. Thanks!

**Comment by henkiewie** [comment]
No it's also missing from the all All Artist view also. Look at the second screen print. All Artists is showing numbers, but not 10cc they only show up in the search. Album Artists are not showing any numbers at all.

**Comment by michaelherger** [comment]
Would you mind sharing your `library.db` with me?

https://www.dropbox.com/request/T3RctyzGgNg0oFDubq6a

**Comment by michaelherger** [comment]
Got your file. Interesting finding: the "10cc" entry has a `namesort` value of 'HOTLEGS'. You should probably check the tags of that track (and others by 10cc).

**Comment by henkiewie** [comment]
That's something new. I've been looking at my ID3 tags and they have become a bit messy because of all the different editors I used over the years. I think I never filled the tag namesort or any of the other sort tags, but maybe some of them where  already filled I found an additional tag called hotlegs indeed no idea where that came from.
I just decided to clear them all for 10cc and did a clear and rescan and 10cc is back 😄 . Thanks for the help.
![Screenshot 2022-02-14 at 14 54 27](https://user-images.githubusercontent.com/74558874/153877121-ab5b1ae8-d46d-4fb8-8444-a3f0f41f2c59.png)
